docker-compose.yml

```diff
@@ -2,12 +2,23 @@
 
 version: '3.6'
 
+x-logging:
+    &common-logging
```

Comment:
Where does the json-file get created for logs?

Reply:
I had to google this - you can still view the logs using 'docker logs', but it will also store the logs within the docker folders on the local host:

`docker inspect -f {{.LogPath}} <container name>`

Reply:
I should note that for production, this should really be set to 'syslog' or 'journald'.  The 'json-file' option is only there because of mac users.

Reply:
Nice, might want to add to switch to syslog for production systems in the comment.

Reply:
I think it's better to leave this up to the user.  The json-file is a 'safe' option that should work on all platforms, while syslog and journald are distro specific.

Reply:
OK sounds good.